src/styles/services-view.less

```diff
@@ -105,6 +105,17 @@
     }
   }
 
+  .card-expansion {
+    overflow-y: auto;
+    .card-view-pf {
+      max-height: 410px;
+    }
+
+    &.no-sub-cats {
+      margin-top: 12px;
+    }
+  }
```

Comment:
Remove the `overflow-y` and `.card-view-pf max-height` styles from above (line 86) since they're already set here. Shouldn't need to declare them twice.